daemon/src/taker_cfd_actor.rs

```diff
@@ -142,10 +155,34 @@ pub fn new(
 
                         inbox.send(msg).unwrap();
                     }
-                    Command::CfdSetupCompleted(_finalized_cfd) => {
-                        todo!("but what?")
+                    Command::CfdSetupCompleted { order_id, dlc } => {
+                        dbg!("Setup complete, publishing on chain now...");
```

Comment:
Nit: With `dbg` this will essentially print _twice_. String literals in `dbg` are basically always pointless.